tests/enable-gcp-services/idempotent-prune/.expected/results.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kpt.dev/v1
+kind: FunctionResultList
+metadata:
+  name: fnresults
+exitCode: 0
+items:
+  - image: gcr.io/kpt-fn/enable-gcp-services:unstable
+    exitCode: 0
+    results:
```

Comment:
I don't quite follow what is the purpose of this test. Can you please explain? Maybe add a comment some where.

Reply:
Added a [comment](https://github.com/GoogleContainerTools/kpt-functions-catalog/blob/0549f1360eb7df7ffee25401589b022d12a5420e/tests/enable-gcp-services/idempotent-prune/services.yaml#L6). What I am testing is that a given ProjectServiceList prunes any previous service it generated and regenerates new ones. This allows us to manage all the generated services owned by that  ProjectServiceList (like removing an old service, removing any manual changes to a generated service etc).